File: src/test-files/js/nested.js
function firstLevel() {
  console.log("First level");
  function secondLevel() {
    console.log("Second level");
    function thirdLevel() {
      console.log("Third level");
      function fourthLevel() {
        console.log("Fourth level");
      }
      fourthLevel();
    }
    thirdLevel();
  }
  secondLevel();
}
firstLevel();

File: src/test-files/rb/nested.rb
def first_level
  puts "First level"

  def second_level
    puts "Second level"

    def third_level
      puts "Third level"

      def fourth_level
        puts "Fourth level"

        def fifth_level
          puts "Fifth level"
        end

        fifth_level
      end

      fourth_level
    end

third_level
  end

  second_level
end

first_level
